libs/langchain/langchain/retrievers/multi_vector.py

```diff
@@ -27,12 +28,40 @@ class MultiVectorRetriever(BaseRetriever):
     and their embedding vectors"""
     docstore: BaseStore[str, Document]
     """The storage layer for the parent documents"""
-    id_key: str = "doc_id"
-    search_kwargs: dict = Field(default_factory=dict)
+    id_key: str
+    search_kwargs: dict
     """Keyword arguments to pass to the search function."""
-    search_type: SearchType = SearchType.similarity
+    search_type: SearchType
     """Type of search to perform (similarity / mmr)"""
 
+    def __init__(
+        self,
+        *,
+        vectorstore: VectorStore,
+        docstore: Optional[BaseStore[str, Document]] = None,
+        base_store: Optional[BaseStore[str, bytes]] = None,
+        id_key: str = "doc_id",
+        search_kwargs: dict = {},
```

Comment:
Use None as the default as dict is mutable

```suggestion
        search_kwargs: Optional[dict] = None,
```

Then downstream

```
search_kwargs = search_kwargs if search_kwargs is not None else {}
```